### DISCUSSION TITLE
请求不同用户可以选择多个不同的存储

### DISCUSSION THREAD

**Discussion by alexleft** [AlistGo/alist#2944] [Ideas]
目前一个用户只能选择一个根目录，请求新功能：管理员可以配置不同用户使用一个或多个存储构建成该用户的目录，以实现跟细致的权限设置提升。

*有看到其他用户提类似的问题，希望可以得到响应。

**Comment by NovaShen0724** [comment]
感觉需求不大，你不如多把同一个储存挂载到不同的用户的根目录下，麻烦点而已
